### PR TITLE
Add support for SLE Micro 5.4, SLES 15 SP5, and Leap 15.5

### PR DESCRIPTION
**Pull request by jcayouette**
# Description

This PR adds support for:

- SLE Micro 5.4
- SLES 15 SP5
- OpenSUSE Leap 15.5

Later:  PR for Uyuni can be prepared already, but cannot be merged until PR's https://github.com/uyuni-project/uyuni/pull/6872 and https://github.com/uyuni-project/uyuni/pull/6871 are merged.
SLE Micro 5.4
openSUSE Leap Micro 5.4

Which documentation version does this PR apply to?
# Target branches

- [X] Master (Default) Needs to be backported later: see above.
- [X] Manager-4.3
- [ ] Manager-4.2

Merge order, the second PR reverts a few mistakes from the first.

1: https://github.com/uyuni-project/uyuni-docs/pull/2195
2: https://github.com/uyuni-project/uyuni-docs/pull/2198

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Review by keichwa — APPROVED** [review]

File: modules/quickstart/pages/install-uyuni-server.adoc [hunk 1]
```diff
@@ -17,7 +17,7 @@ This table shows the software and hardware requirements for installing an Uyuni
 .Software and Hardware Requirements
 |===
 | Software and Hardware  | Recommended
-| Operating System:      | openSUSE Leap 15.4: Clean installation, up-to-date
+| Operating System:      | openSUSE Leap 15.5: Clean installation, up-to-date
```
Comment (admd) [inline modules/quickstart/pages/install-uyuni-server.adoc:20]: nope, same as above...currently it needs to run on top of 15.4

File: modules/quickstart/pages/quickstart-install-uyuni-server.adoc [hunk 2]
```diff
@@ -17,7 +17,7 @@ This table shows the software and hardware requirements for installing an Uyuni
 .Software and Hardware Requirements
 |===
 | Software and Hardware  | Recommended
-| Operating System:      | openSUSE Leap 15.4: Clean installation, up-to-date
+| Operating System:      | openSUSE Leap 15.5: Clean installation, up-to-date
```
Comment (admd) [inline modules/quickstart/pages/quickstart-install-uyuni-server.adoc:20]: same as above

File: modules/installation-and-upgrade/pages/db-migration-xy.adoc [hunk 3]
```diff
@@ -6,8 +6,8 @@ If you are already using PostgreSQL 14, you do not need to perform this migrati
 
 If you want to upgrade to the latest {productname} version, you must be using PostgreSQL version 13 or 14, depending on the underlying operating system:
 
-* If you are running SLES 15 SP4, use PostgreSQL 14.
-* If you are running {opensuse} Leap 15.4, use PostgreSQL 14.
+* If you are running SLES 15 SP5, use PostgreSQL 14.
+* If you are running {opensuse} Leap 15.5, use PostgreSQL 14.
```
[the remark this answers is not in the document]
Comment (jcayouette) [reply modules/installation-and-upgrade/pages/db-migration-xy.adoc:10]: reverted

**Comment by jcayouette** [comment]
All comments fixed.

**Pull request merged**